scripts/kind-e2e/cluster3-config.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  networking:
+    podSubnet: 10.246.0.0/16
+    serviceSubnet: 100.96.0.0/16
+    dnsDomain: cluster3.local
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
```

Comment:
... same here

Reply:
Yes this this is kind configuration for 2 worker nodes.